src/Domain/DomainHelpers.cpp

```diff
@@ -582,14 +582,24 @@ std::array<OrientationMap<3>, 6> orientations_for_wrappings() {
 }
 
 namespace {
-size_t which_wedge_index(const ShellWedges& which_wedges) {
+std::array<size_t, 6> which_wedge_indices(const ShellWedges& which_wedges) {
   switch (which_wedges) {
     case ShellWedges::All:
-      return 0;
-    case ShellWedges::FourOnEquator:
-      return 2;
+      return {1, 1, 1, 1, 1, 1};
+    case ShellWedges::AllAndHalvesXY:
+      return {2, 2, 2, 2, 1, 1};
+    case ShellWedges::AllAndHalvesXZ:
+      return {2, 2, 2, 2, 1, 1};
+    case ShellWedges::AllAndHalvesYZ:
+      return {2, 2, 2, 2, 1, 1};
```

Comment:
The AllAndHalves cases are all rotated versions of the AllAndHalvesYZ because half wedges are created by splitting the Wedge along the xi direction, and the wedges in the YZ plane are the only wedges that have their xi axes parallel to each other (so that block boundary created by splitting the wedges is an equatorial circle as opposed to various disconnected arcs) (I will add a comment to this effect :slightly_smiling_face: )